pkg/config/constants/constants.go

```diff
@@ -39,7 +39,7 @@ const (
 	// DefaultRootCert is the default path to the mTLS root cert file
 	DefaultRootCert = AuthCertsPath + RootCertFilename
 
-	// ConfigPathDir config directory for storing envoy json config files.
+	// ConfigPathDir config directory for storing envoy json config files and downloaded Wasm modules.
 	ConfigPathDir = "./etc/istio/proxy"
```

Comment:
nit: in the future, should we use another directory to store the wasm modules? traditionally, `/etc/` isn't really where you expect binary blobs or caches, something under `/var/` would probably be more appropriate. but that would require changes to our charts. possibly a follow-up ticket?

Reply:
Yeah, good point. We actually already had a folder `/var/lib/istio/data`, which is used to store core dump and profile. Wasm module is more appropriate there. I updated the code to use that instead.